tests/lib/prepare.sh

```diff
@@ -96,29 +101,53 @@ prepare_classic() {
     if snap --version |MATCH unknown; then
         echo "Package build incorrect, 'snap --version' mentions 'unknown'"
         snap --version
-        apt-cache policy snapd
+        case "$SPREAD_SYSTEM" in
```

Comment:
Could we abstract this to pkgdb.sh as well maybe? Feels a bit ugly to have this test here.